trackintel/model/util.py

```diff
@@ -208,21 +157,23 @@ def _constructor_with_fallback(*args, **kwargs):
             if isinstance(result, GeoDataFrame):
                 # cannot validate geometry as geometry column is maybe not set
                 return self.__class__(result, validate_geometry=False)
```

Comment:
To clarify: this then means the super constructor is a GeoDataFrame, but why for the self class we do not check the geometry? For a class to be the type of GeoDataFrame, you need to set the geometry column, no?

Reply:
A GeoDataFrame without **set** geometry is possible. In our `_constructor` we create a new GeoDataFrame without set geometry, thus we cannot validate the geometry, as we cannot access `gdf.geometry`.
The geometry column will only be set, if we get back to the geopandas function that created the `_constructor` call. One example for this would be the GeoDataFrame method [`merge`](https://github.com/geopandas/geopandas/blob/0eb2a5ecdc3b7b595e406c9f7bf1e6435ad39828/geopandas/geodataframe.py#L1556).

Reply:
Thanks for the example. This is clear now